CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the forked repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
```

Comment:
nit: Should it be `OpenShift` or `Openshift`?

Reply:
@SamiSousa looks like it's OpenShift :)